day_6/exercises/person.rb

```diff
@@ -1,5 +1,35 @@
-# Create a person class with at least 2 attributes and 2 behaviors. 
+# Create a person class with at least 2 attributes and 2 behaviors.
 # Call all person methods below the class and print results
 # to the terminal that show the methods in action.
 
 # YOUR CODE HERE
+
+class Person
+  attr_reader :name, :age, :height
+  def initialize(name, age, height)
+    @name = name
+    @age = age
+    @height = height
+  end
+
+  def add_heel_lifts(new_height)
+    puts "adding heel lifts of #{new_height} cm"
+    @height = @height + new_height
+  end
+
+  def celebrate_birthday(new_age)
+    @age = @age + 1
+    puts "celebrating birthday, new age is #{@age}"
```

Comment:
Since you do not use your parameter `new_age` inside of this method, you don't really need one! You can run this method without needing a parameter... seems like it will add 1 to an instances age no matter what their current age is.